File: cpp/Leetcode/c++/problem2864.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    string maximumOddBinaryNumber(string s) {
        int o = count(s.begin(),s.end(),'0');
        int e= count(s.begin(),s.end(),'1');
        string res;
        for(int i=0;i<e-1;i++){
            res+='1';
        }
        for(int i=0;i<o;i++){
            res+='0';
        }
        res+='1';
        return res;
    }
};

File: cpp/Leetcode/Python/problem2864.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        c=s.count('1')
        d=s.count('0')
        t=''
        for i in range(c-1):
            t=t+'1'
        for j in range(d):
            t=t+'0'
        t=t+'1'
        return t